documentation/book/appendix_metrics.adoc

```diff
@@ -14,89 +14,235 @@ endif::InstallationAppendix[]
+When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the Alertmanager which then uses the configured notification methods to notify the user.
+
+For more information about setting up alerting rules, see https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[Alerting Rules] in the Prometheus documentation.
+
 ifdef::Kubernetes[]
-==== Deploying on {KubernetesName}
+==== Deploying Prometheus on {KubernetesName}
 
 The provided `prometheus.yaml` file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+It also discovers an Alertmanager instance in the same namespace.
 If you are using a different namespace, download the resource file and update it as follows:
 
-[source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus.yaml
+On Linux, use:
+
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+
+On MacOS, use:
+
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+
+
+To define Prometheus jobs that will scrape the metrics data, you must apply the `ServiceMonitor` resource located in the provided `strimzi-service-monitor.yaml` file.
+Download it by using the following command:
+
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/strimzi-service-monitor.yaml
 
-The provided `alerting-rules.yaml` file creates a `ConfigMap` with sample alerting rules; download the resource file as follows:
+Currently, the Prometheus Operator only supports jobs that include an `endpoints` role for service discovery. To use another role, edit the `additionalScrapeConfigs` property in the `prometheus.yaml` configuration file. This takes the name of the `Secret` and the name of the property in a given `Secret` in which additional configuration is stored.
 
-[source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml > alerting-rules.yaml
+To create this `Secret` resource, use the following command:
 
-To deploy all these resources you can run the following.
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/additional-properties/prometheus-additional.yaml
```

Comment:
Same here, the file is not downloaded, but written to STDOUT, I would add a -O (and probably remove the -s) option